spec/models/publication_spec.rb

```diff
@@ -336,31 +336,28 @@
 
     it 'should refuse to add a publication with the same source record' do
       save_new_publication
-      expect do
-        Publication.build_new_manual_publication(pub_hash, 'some string', 'some where')
-      end.to raise_error(ActiveRecord::RecordNotUnique)
+      expect { Publication.build_new_manual_publication(pub_hash, 'some string') }.to raise_error(ActiveRecord::RecordNotUnique)
     end
```

Comment:
The line was probably broken to avoid rubocop long line issue, plus it's possible to see everything in a github PR; not it's not all visible in this PR as one long line.